code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -485,6 +484,55 @@
 
 		to_chat(M, "<span class='danger'><font size=3>You have been kidnapped and interrogated for valuable information! You will be sent back to the station in a few minutes...</font></span>")
 
+
+/**
+  * Handles the target's injury/interrogation at the Syndicate Jail.
+  *
+  * Arguments:
+  * * M - The target mob.
+  */
+/datum/syndicate_contract/proc/injure_target(mob/living/M)
+	if(prob(RETURN_INJURY_CHANCE) && M.health >= 50)
+		var/obj/item/organ/external/injury_target
+		var/do_damage = FALSE
+		if(prob(20)) //remove a limb
+			if(prob(50))
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+				if(!injury_target)
+					do_damage = TRUE
+					return
+				injury_target.droplimb()
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! Oh god something's missing!</span>")
+		else //fracture
+			if(ismachineperson(M))
+				M.emp_act(EMP_HEAVY)
+				M.adjustBrainLoss(30)
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like some of your components are loose!</span>")
+
+			else if(isslimeperson(M))
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+				if(!injury_target)
+					do_damage = TRUE
+					return
+				injury_target.cause_internal_bleeding()
+
+				injury_target = M.get_organ(BODY_ZONE_CHEST)
+				injury_target.cause_internal_bleeding()
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like your inner membrane has been punctured!</span>")
+
+			else if(prob(10))
+				injury_target = M.get_organ(BODY_ZONE_CHEST)
+				injury_target.fracture()
+			else
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_R_LEG, BODY_ZONE_R_LEG))
+				if(!injury_target)
+					do_damage = TRUE
+					return
+				injury_target.fracture()
+		if(do_damage)//if no limb is found, damage them..
+			M.adjustBruteLoss(40)
+			M.adjustBrainLoss(25)
```

Comment:
Might just want to make this into a proc which you call instead of setting `do_damage = TRUE`

If you do this, then you can also place early returns after each case. So you don't need an `else if` chain.